.nancy-ignore

```diff
@@ -1,4 +1,4 @@
-# The nancy command has found a vulnerability, which is an out of date version of GoGo Protobuf. However, this cannot be updated due to the project also having an indirect dependency of something called 
+# The nancy command has found a vulnerability, which is an out of date version of GoGo Protobuf. However, this cannot be updated due to the project also having an indirect dependency of something called
```

Comment:
Error with "make audit" needs to be fixed.... 
go list -json -m all | nancy sleuth
/bin/sh: nancy: command not found
make: *** [audit] Error 127

Reply:
This error only occurs locally, I think it works ok if nancy is installed, the audit checks pass in concourse.